app/actions/views/login.js

```diff
@@ -70,26 +71,25 @@ export function getSession() {
     return async (dispatch, getState) => {
         const state = getState();
         const {currentUserId} = state.entities.users;
-        const {credentials} = state.entities.general;
-        const token = credentials && credentials.token;
+        const {deviceToken} = state.entities.general;
```

Comment:
Can we use `getCurrentUserId` selector?
For `deviceToken`, we might need to have redux selector like `getDeviceToken` but maybe good for improvement to `master`.

Reply:
There is no computation needed so I don’t see why we need a selector, and btw getCurrentUserId is just a function, getCurrentUser is in fact a selector ;)